src/config_file.c

```diff
@@ -12,6 +12,9 @@ G_DEFINE_QUARK(r-slot-error-quark, r_slot_error)
 
 #define RAUC_SLOT_PREFIX	"slot"
 
+#define UBOOT_DEFAULT_ATTEMPTS  3
+#define UBOOT_ATTEMPTS_PRIMARY  3
```

Comment:
For keeping the bootloader implementation more central, I would propose letting these values reside in `bootchooser.c` and perform the fallback to default values in `uboo_env_set()`.

Reply:
Hmm, and then look at the key again in `uboot_set_env` to infer which variable is being written?

Reply:
I noted this under my wrong assumption that we have a method for explicitly writing the attempts only. I still would prefer having the defaults in the bootloader implementation. One could apply them conditionally when setting the `val =` we had in the discussion above. What do you think?

Reply:
Yes, makes sense. Did that now. PTAL!